js/bibox.js

```diff
@@ -452,6 +452,11 @@ module.exports = class bibox extends Exchange {
     }
 
     parseOrder (order, market = undefined) {
+        let keys = Object.keys (order);
+        let numKeys = keys.length;
+        if (numKeys < 1) {
```

Comment:
Sorry, my bad, just edited it again to make it pass )

Reply:
I'll add a `isEmpty` base method for this a little later as this feels like something common across all exchanges.

Reply:
no problem, thanks